AlayaCare Cloud Tech Radar.csv

```diff
@@ -15,26 +15,25 @@
 "Adopt","Python 3","languages-and-frameworks","FALSE","<p></p>"
-"Adopt","Vue 3","languages-and-frameworks","TRUE","<p>Vue 3 is the prefered language for new front-end developement.</p>"
-"Adopt","TypeScript","languages-and-frameworks","TRUE","<p>TypeScript is the prefered language for new front-end developement.</p>"
+"Adopt","Vue 3","languages-and-frameworks","TRUE","<p>Vue 3 is the prefered language for new frontend developement.</p>"
+"Adopt","TypeScript","languages-and-frameworks","TRUE","<p>TypeScript is the prefered language for new frontend developement.</p>"
+"Adopt","single-spa","languages-and-frameworks","FALSE","<p><a href='https://single-spa.js.org/'>single-spa</a> is a JavaScript framework for bringing together multiple micro frontends in a single frontend application.</p>"
 "Adopt","Micro frontend","Techniques","FALSE","<p></p>"
 "Adopt","Backstage","Platforms","FALSE","<p>Backstage is the source of truth for our component's scorecards</p>"
 "Adopt","Snyk.io","Platforms","FALSE","<p><a href='https://snyk.io/'>snyk.io</> - Automatically detect vulnerabilities</a></p>"
+"Adopt","Module Federation","languages-and-frameworks","FALSE","<p><a href='https://webpack.js.org/concepts/module-federation/'>Module federation</a> is infrastructure-level solution for developing micro frontends provided by Webpack</p>"
 "Trial","GitOps","Techniques","FALSE","<p>GitOps is a way of implementing Continuous Deployment for cloud native applications. It focuses on a developer-centric experience when operating infrastructure, by using tools developers are already familiar with, including Git and Continuous Deployment tools.</p>"
 "Trial","Tech Radars","Techniques","FALSE","<p></p>"
+"Trial","Vitest","Techniques","FALSE","<p>JavaScript testing framework with the potential to replace Jest.</p>"
 "Assess","Accessibility","languages-and-frameworks","FALSE","<p></p>"
-"Assess","Module Federation","languages-and-frameworks","FALSE","<p><a href='https://webpack.js.org/concepts/module-federation/'>Module federation</a> is infrastructure-level solution for developing micro frontends provided by Webpack</p>"
-"Assess","Rollup","languages-and-frameworks","FALSE","<p><a href='https://rollupjs.org/guide/en/'>Rollup</a> is a module bundler for JavaScript which compiles small pieces of code into something larger and more complex, such as a library or application. It uses the new standardized format for code modules included in the ES6 revision of JavaScript, instead of previous idiosyncratic solutions such as CommonJS and AMD. ES modules let you freely and seamlessly combine the most useful individual functions from your favorite libraries. This will eventually be possible natively everywhere, but Rollup lets you do it today.</p>"
 "Assess","Vault","Tools","FALSE","<p></p>"
 "Hold","MariaDB","Platforms","FALSE","<p></p>"
-"Hold","Bootstrap","languages-and-frameworks","FALSE","<p></p>"
```

Comment:
I don't see bootstrap anywhere, don't we want to keep a Hold do mention that it should not be used and to use XXX instead (ADS?)

Reply:
This is a fair point that I discussed with Rob. We want to keep the radar hold section small and relevant so that people read the blips. 

He explained that bootstrap has been out of use for some time and isn`t actively discussed. Bringing it back should trigger a discussion anyway because it is not in the current version of the radar.